_posts/photography/2015-03-10-my_gear.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: post_layout
+title: My photography gear
+avatar: 
+category: Photography
+tags: [Photography - Gear - Cameras - Bronica GS-1 - Contax G1 - Olympus XA2 - Olympus OM2n - Olympus OMD EM-5]
+path: posts
+---
+
+The second most popular question following the "Why you shoot film?" is **"What gear you own and use?"**
+
+So this is the list of all the gear I own and use!
+
+## **Bronica GS-1**
+with **Zenzanon PG 100mm f/3.5** and **Zenzanon PG 65mm f/4**  
+
+this is my main camera, the one that I carry with me when I go out to photograph, from landscapes to portraits. 
+
+I am very happy with the camera as it produces very good results,
+it has good metering with the AE prism and sharp lenses.
+
+**Tip:**
+*Focusing improved dramatically when I changed the regular focusing screen with a split-screen. I recommend this upgrade to anyone with a GS-1.*
+
+The built quality is also very good, solid with good ergonomics witch makes the use of the camera pretty straightforward.
+
+The Zenzanon PG series of lenses are very good, sharp across the field, even wide open and can be found in very reasonable prices. 
+
+I use mostly the *PG 65mm* due to the style of my work and the *100mm* when I want to shoot portraits. 
```

Comment:
I mostly use the *PG 65mm* due to my work style and...
Due to my work style (urban landscapes) I mostly use the *PG 65mm* and whenever I shoot portraits I use the *100mm* one.
Γενικά το "work style" νομιζω δεν εκφραζει αυτο που θες να πεις, θα σου πω εναλλακτικη για ολη την προταση οταν εχω χρονο